q2_feature_table/_summarize/_visualizer.py

```diff
@@ -272,7 +274,6 @@ def _frequencies(table, axis):
 
 def _frequency_summary(table, axis='sample'):
     frequencies = _frequencies(table, axis=axis)
-
```

Comment:
I would put this new line back in just to keep the diff a bit cleaner, since it's not a part of this PR's functionality.